servatrice/servatrice.ini.example

```diff
@@ -107,6 +107,14 @@ allowedpunctuation=_.-
 ; If a username can begin with punctuation defined in allowedpunctuation
 allowpunctuationprefix=false
 
+; Disallow usernames containing these words. This list is comma seperated
+disallowedwords="admin"
```

Comment:
Also, please add an example for multiple items: is it `"word1", "word2"` or `"word1, word2"` ? As an user i'd have to look it up or test it to be sure, better to make it clear in the comment.